src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -63,6 +63,9 @@ public function block($messages, $type = null, $style = null, $prefix = ' ', $pa
         $messages = array_values((array) $messages);
```

Comment:
I suggest replacing this strange code and the patch below with:
`$messages = is_array($messages) ? array_values($messages) : array($messages);`